go/chat/inboxsource.go

```diff
@@ -40,12 +56,14 @@ type RemoteInboxSource struct {
 
 func NewRemoteInboxSource(g *libkb.GlobalContext, boxer *Boxer, ri func() chat1.RemoteInterface,
 	tlf func() keybase1.TlfInterface) *RemoteInboxSource {
+	udc := utils.NewUserDeviceCache(g)
```

Comment:
It's not, let me see if I can approve that and I can integrate that change here.